app/src/main/java/com/sun/makeupwindow/data/source/local/dao/ColorDAO.kt

```diff
@@ -0,0 +1,8 @@
+package com.sun.makeupwindow.data.source.local.dao
+
+import com.sun.makeupwindow.data.model.Color
+
+interface ColorDAO {
```

Comment:
```suggestion
interface ColorDao {
```
đã cmt pull trc r thì pull sau đừng bị lại lỗi đó !

Reply:
dạ, em sẽ rút kinh nghiệm